dom/nodes/insertion-effects/Node-appendChild-script-and-stylesheet-link-from-fragment.tentative.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting script and stylesheet link from a fragment</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+<script>
+const happened = [];
+const link = document.createElement("link");
+test(() => {
+  link.rel = "stylesheet";
+  link.href = "data:text/css,"
+
+  const script = document.createElement("script");
+  script.textContent = `
+    happened.push(link.sheet === null);
+`;
+
+  const df = document.createDocumentFragment();
+  df.appendChild(script);
+  df.appendChild(link);
+
+  assert_array_equals(happened, []);
+  document.body.appendChild(df);
+  assert_array_equals(happened, [true]);
```

Comment:
This is interesting, does Chromium fail this test? And if so, why is this the right behavior? When I do the following in Chromium:

```js
const script = document.createElement('script');
script.textContent = `
  console.log(link.sheet);
`;

const link = document.createElement('link');
link.rel = 'stylesheet';
link.href = 'data:text/css,';

const fragment = new DocumentFragment();
fragment.append(script, link);
document.head.append(fragment);
```

The script observes a non-null sheet, which I think is what I would expect. Is that wrong?